packages/core/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -14,10 +14,10 @@ export class TdsBreadcrumbs {
   render() {
     this.host.children[this.host.children.length - 1]?.classList.add('last');
     return (
-      <nav>
-        <div role="list" class={'tds-breadcrumb'}>
+      <nav aria-label="Breadcrumb">
+        <ol>
           <slot></slot>
-        </div>
+        </ol>
```

Comment:
Have you tested this in terms of valid DOM nesting? 

I think you will up with this in the DOM:
```html
<tds-breadcrumbs>
   // shadow DOM
   <ol>
     <tds-breadcumb>
      // shadow DOM
         <li>
            // slotted content
         </li>
       </tds-breadcumb>
   </ol>
</tds-breadcrumbs>
```

Which will mean the the li are not direct descendants of the ol.
<img width="765" alt="Screenshot 2025-03-06 at 14 21 55" src="https://github.com/user-attachments/assets/4c3b59dc-285b-4ff2-899e-6ffe267f7ae6" />